app/models/upload_reference.rb

```diff
@@ -6,6 +6,10 @@ class UploadReference < ActiveRecord::Base
 
   delegate :to_markdown, to: :upload
 
+  def targets?(target_to_check)
```

Comment:
Wondering if we should name this `#target?` to follow Ruby conventions, e.g. `Enumberable#include?`. Avoids ambiguity of whether "targets" is plural of "target".

Reply:
Yes okay I will change :+1: